### ISSUE TITLE
Annotator syntax inconsistency

### ISSUE THREAD

**Issue by vargenau** [spdx/spdx-spec#683]
Section 12.1 of the ISO standard

Format is:
```
"Person: person name" and optional  "(email)"
```

An example given is:
```
<annotator> Person: Jane Doe () </annotator>
```

If  "(email)" was optional, I would have expected:
```
<annotator> Person: Jane Doe </annotator>
```

Can this be clarified? are the () mandatory?

**Comment by goneall** [comment]
> Can this be clarified? are the () mandatory?

My interpretation was the entire string including the parenthesis is optional.

I just checked and the Java tools do not require the parenthesis.

Unless anyone objects, I would recommend removing the `()` from the example.

**Comment by vargenau** [comment]
Hi,

Thank you for your answer.

I have created a pull request:
https://github.com/spdx/spdx-spec/pull/685

Marc-Etienne

**Comment by goneall** [comment]
Resolved with PR #684 

Thanks @vargenau
